packages/dd-trace/src/plugins/util/url.js

```diff
@@ -0,0 +1,26 @@
+const { URL } = require('url')
+
+function filterSensitiveInfoFromRepository (repositoryUrl) {
+  if (!repositoryUrl) {
+    return ''
+  }
+  if (repositoryUrl.startsWith('git@')) {
+    return repositoryUrl
+  }
+
+  // Remove the username from ssh URLs
+  if (repositoryUrl.startsWith('ssh://')) {
+    const sshRegex = /^(ssh:\/\/)[^@/]*@/
+    return repositoryUrl.replace(sshRegex, '$1')
+  }
+
+  try {
+    const { protocol, hostname, pathname } = new URL(repositoryUrl)
+
+    return `${protocol}//${hostname}${pathname === '/' ? '' : pathname}`
```

Comment:
missing colon no ?
```suggestion
    return `${protocol}://${hostname}${pathname === '/' ? '' : pathname}`
```

Reply:
`protocol` includes the colon: 
```
> url = new URL('http://www.google.com')
URL {
  href: 'http://www.google.com/',
  origin: 'http://www.google.com',
  protocol: 'http:',
  username: '',
  password: '',
  host: 'www.google.com',
  hostname: 'www.google.com',
  port: '',
  pathname: '/',
  search: '',
  searchParams: URLSearchParams {},
  hash: ''
}
```

Reply:
Interesting, when you set the prop, it doesn't need the colon:
![image](https://github.com/DataDog/dd-trace-js/assets/70517128/c53c6143-333e-447a-b5aa-263a514c33d0)